src/pages/index.tsx

```diff
@@ -85,30 +92,30 @@ const HomePage = () => (
             </Heading>
             <List fontSize="" color="secondaryTextLightmode" spacing="4">
               <ListItem mt="-2">
-                <ListIcon as={FaCheck} color="sky.400" />
+                <ListIcon as={HiOutlineSwitchHorizontal} color="sky.400" />
```

Comment:
The outline Heroicons are at their best at `24px` by `24px`. They are not intended to be shrunk/grown.

Right now they're displayed at `16px` by `16px` on our homepage feature list, can we fix this in a later PR? We need to change the lists to a more "feature highlight" display, as designed in Figma later on. 😉 

I like the look of the icons! It's a massive improvement over the generic check-marks! 😉

Reply:
Updated so they take 24px here. All the positioning, background circle, etc will be addressed in a separate PR tho.

Looking really good!